### ISSUE TITLE
Questions about binary classification task

### ISSUE THREAD

**Issue by yangzhao-666** [facebookresearch/LearningToLearn#8]
Hi there,

Really thanks for your work. Very clear and readable code, very interesting idea and experiments. I'm currently reproducing the binary classification tasks, and I was wondering what the input for ml3 loss is in classification tasks, is it the same as for crossEntropy loss? or some other engineering techniques are used for the input?

Thank you very much and looking forward to your reply.

**Comment by fmeier** [comment]
Hi, 
thank you!

For the binary classification results in the paper the learned loss function takes in the predicted and ground truth target, just like the cross entropy loss.

Hope this helps.

**Comment by yangzhao-666** [comment]
> Hi,
> thank you!
> 
> For the binary classification results in the paper the learned loss function takes in the predicted and ground truth target, just like the cross entropy loss.
> 
> Hope this helps.

**Comment by yangzhao-666** [comment]
> Hi,
> thank you!
> 
> For the binary classification results in the paper the learned loss function takes in the predicted and ground truth target, just like the cross entropy loss.
> 
> Hope this helps.

Hi Franziska, 

Thanks for getting back so quickly. This definitely helps. Can I ask one more question?  Could you please comment on the extension of ml3 loss on multi-class classification tasks? Do you think it might also work?

Thanks for your time and looking forward to your reply.

Best,
Zhao

**Comment by fmeier** [comment]
Hi Zhao,

in theory it should work for multi-class classification tasks!

Best,
Franziska

**Comment by yangzhao-666** [comment]
Hi Franziska,

Thank you very much for the reply! Very helpful!

Best,
Zhao

**Comment by mikehuisman** [comment]
> Hi,
> thank you!
> 
> For the binary classification results in the paper the learned loss function takes in the predicted and ground truth target, just like the cross entropy loss.
> 
> Hope this helps.

Hi Franziska, 

Did you apply a softmax operation over the predictions before passing them to the meta-loss network? Or did you pass in the raw predictions?

Kind regards,

Mike
